packages/yourloops/test/integration/pages/patient-dashboard/patient-dashboard.common.spec.tsx

```diff
@@ -107,7 +107,7 @@ describe('Patient dashboard for anyone', () => {
     })
     const statsWidgets = await screen.findByTestId('patient-statistics', {}, { timeout: 3000 })
     expect(statsWidgets).toBeVisible()
-    expect(statsWidgets).toHaveTextContent('Time In Range2h8%10h42%6h25%4h17%2h8%<5454-7070-180180-250>250mg/dLStandard Deviation (61-209)mg/dL74Avg. Glucose (CGM)mg/dL135Sensor Usage1%CV (CGM)55%Avg. Daily Total Insulin(1.3U)Bolus1.3 U100%Basal0.0 U2%Avg. Daily Insulin1.3UWeight72kgDaily Dose ÷ Weight0.02U/kgAvg. Daily Time In Loop ModeONOFF91%21h 49m9%2h 11mAvg. Daily Carbs55gRescue carbs10g')
+    expect(statsWidgets).toHaveTextContent('Time In Range2h8%10h42%6h25%4h17%2h8%<5454-7070-180180-250>250mg/dLStandard Deviation (61-209)mg/dL74Avg. Glucose (CGM)mg/dL135Sensor Usage1%CV (CGM)55%Avg. Daily Total Insulin(1.3U)Bolus1.3 U100%Basal0.0 U2%Avg. Daily Insulin1.3UWeight72kgDaily Dose ÷ Weight0.02U/kgAvg. Daily Time In Loop ModeONOFF91%21h 49m9%2h 11mAvg. Daily Carbs8gRescue carbs1g')
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** This change does not feel right. How comes that we don't have the same result than before?‏